docs/supported_docker_environment/continuous_integration/gitlab_ci.md

```diff
@@ -1,5 +1,37 @@
 # GitLab CI
 
+## example with docker.sock
+If you have your own docker runner installed, and you have `/var/run/docker.sock` mounted in the gitlabrunner.
+
+See below for example configuration for your own docker gitlab runner: 
+```toml
+[[runners]]
+  name = "MACHINE_NAME"
+  url = "https://gitlab.com/"
+  token = "GENERATED_GITLAB_RUNNER_TOKEN"
+  executor = "docker"
+  [runners.docker]
+    tls_verify = false
+    image = "docker:latest"
+    privileged = false
+    disable_entrypoint_overwrite = false
+    oom_kill_disable = false
+    disable_cache = false
+    volumes = ["/var/run/docker.sock:/var/run/docker.sock", "/cache"]
+    shm_size = 0
+```
+
+Please include the following in the test task for `.gitlab-ci.yml`:
+```yml
+variables:
+  TESTCONTAINERS_HOST_OVERRIDE: "host.docker.internal"
+```
+
+The environment variable `TESTCONTAINERS_HOST_OVERRIDE` needs to be configured otherwise a wrong ip address would be 
+assigned to resolve the host of the runner, which could lead to failing tests.
```

Comment:
```suggestion
The environment variable `TESTCONTAINERS_HOST_OVERRIDE` needs to be configured, otherwise, a wrong IP address would be used to resolve the Docker host, which will likely lead to failing tests.
```